project/src/store/reducer.ts

```diff
@@ -1,31 +1,46 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { sortValues } from '../utils/utils';
 import { Cities } from '../const';
-import { City, Offers } from '../types/types';
-import { setSortName, setOffers, switchCity } from './action';
+import { City, OfferType, Reviews } from '../types/types';
+import { setSortName, switchCity, setOffers, setLoadingStatus, setNearbyOffers, setComments } from './action';
 
 type ReducerTypes = {
   currentCity: City;
-  offersList: Offers;
   currentSorting: string;
+  offers: { [offerId: number]: OfferType};
+  commentsList: Reviews;
+  isLoading: boolean;
+  nearbyOffersList: { [offerId: number]: OfferType};
```

Comment:
офферы поблизости мы каждый раз перефетчиваем при переходе на страницу оффера

простого списка здесь достаточно, объект не нужен
```suggestion
  nearbyOffersList: OfferType[];
```